Index: rutorrent/conf/config.php
===================================================================
--- rutorrent.orig/conf/config.php
+++ rutorrent/conf/config.php
@@ -19,7 +19,7 @@
 	$schedule_rand = 10;			// rand for schedulers start, +0..X seconds
 
 	$do_diagnostic = true;
-	$log_file = '/tmp/errors.log';		// path to log file (comment or leave blank to disable logging)
+	$log_file = '/var/log/rutorrent/rutorrent.log';		// path to log file (comment or make empty to disable logging)
 
 	$saveUploadedTorrents = true;		// Save uploaded torrents to profile/torrents directory or not
 	$overwriteUploadedTorrents = false;     // Overwrite existing uploaded torrents in profile/torrents directory or make unique name
@@ -27,17 +27,16 @@
 	$topDirectory = '/';			// Upper available directory. Absolute path with trail slash.
 	$forbidUserSettings = false;
 
-	$scgi_port = 5000;
-	$scgi_host = "127.0.0.1";
+	//$scgi_port = 5000;
+	//$scgi_host = "127.0.0.1";
 
 	// For web->rtorrent link through unix domain socket 
 	// (scgi_local in rtorrent conf file), change variables 
 	// above to something like this:
-	//
-	// $scgi_port = 0;
-	// $scgi_host = "unix:///tmp/rpc.socket";
+	$scgi_port = 0;
+	$scgi_host = "unix:///var/run/rutorrent/rtorrent.sock";
 
-	$XMLRPCMountPoint = "/RPC2";		// DO NOT DELETE THIS LINE!!! DO NOT COMMENT THIS LINE!!!
+	$XMLRPCMountPoint = "/rutorrent/RPC";		// DO NOT DELETE THIS LINE!!! DO NOT COMMENT THIS LINE!!!
 
 	$pathToExternals = array(
 		"php" 	=> '',			// Something like /usr/bin/php. If empty, will be found in PATH.
@@ -61,4 +60,4 @@
 
 	$canUseXSendFile = true;		// Use X-Sendfile feature if it exist
 
-	$locale = "UTF8";
\ No newline at end of file
+	$locale = "UTF8";
